src/pages/signUp/signUpSteps/Verification.tsx

```diff
@@ -0,0 +1,71 @@
+import { FunctionComponent, useState } from "react";
+import { FilledButton, GradientTypography, Link, Typography } from "elements";
+import { TextInputField } from "components";
+import { FormikValues, useFormikContext } from "formik";
+import { useDispatch } from "react-redux";
+import { Box, Stack } from "@mui/material";
+import { sendVerificationEmail } from "../utils";
+
+
+const Verification: FunctionComponent = () => {
+  const dispatch = useDispatch();
+  const [showResendLink, setShowResendLink] = useState(true);
+  const { isValid, values } = useFormikContext();
+
+
+  const handleEmailResend = () => {
+
+    setShowResendLink(false);
+
+    dispatch(sendVerificationEmail(values as FormikValues));
+
+    setTimeout(() => setShowResendLink(true), 10000);
+  };
+
+  return (
+    <Box display="flex" flexDirection="column" alignItems="center">
+      <Typography
+        align="center"
+        fontFamily="Raleway"
+        fontWeight="extra-bold"
+        marginBottom="12px"
+        variant="xxxl"
+      >
+        Check your email!
+      </Typography>
+      <GradientTypography
+        align="center"
+        fontFamily="DM Serif Text"
+        fontStyle="italic"
+        fontWeight="regular"
+        id="verificationLabel"
+        marginBottom="60px"
+        variant="xxxl"
+      >
+        Paste your verification code here.
+      </GradientTypography>
+      <Stack spacing={ 1.5 } mb={ 7.5 } maxWidth="312px" width="100%">
+        <TextInputField
+          aria-labelledby="verificationLabel"
+          name="authCode"
+          placeholder="Verification Code"
+          type="text"
+        />
+        <FilledButton type="submit" disabled={ !isValid }>
+          Enter
+        </FilledButton>
+      </Stack>
+
+      <Box alignSelf="center" position="absolute" bottom="32px" left="0" right="0" color="grey200">
```

Comment:
Looks like the absolute positioning is causing the text to be relative to the bottom of the screen, which may cause issues with the responsive layout.

<img width="480" alt="Screen Shot 2022-05-03 at 4 34 28 PM" src="https://user-images.githubusercontent.com/5877597/166599790-6a75b49f-1730-4cb3-b27d-a98530be9e67.png">

Reply:
So this is the reason why i asked for responsive font sizes... bc in the mock it looks like it's absolute positioned to the bottom which works fine for large screens but not so much for smaller screens. I think this shouldn't be an issue once we implement the responsive font sizes... Do you think i should take a different approach?

Reply:
Hmm... well, if we're confident that no one's device will see it overlapping, either way sounds fine to me. Could do something like this to maintain it's presence at the bottom of the screen without absolute positioning it though:

```
<div style={{ minHeight="100vh" display: "flex"; flexDirection: "column", alignContent: "space-between">
  <div>
    // main content here
  </div>

  <div>
    // content at bottom of screen here
  </div>
</div>
``

Reply:
Doing it this way would mean i need to reconsider the layout of the signup to not include the logo and go back to repeating the usage 🤔 I will keep this in mind when i do the responsive story... i think i will be making the change though